pygeoapi/flask_app.py

```diff
@@ -144,6 +179,12 @@ def describe_processes(name=None):
 
 @APP.route('/processes/<name>/jobs', methods=['GET', 'POST'])
 def execute_process(name=None):
+    """
+    OGC open api jobs from processes access point (experimental)
+
```

Comment:
`:param name: identifier of process to execute`

Reply:
Fixed